x-pack/plugins/apm/public/components/app/transaction_details/waterfall_with_summary/waterfall_container/waterfall/span_flyout/index.tsx

```diff
@@ -164,113 +164,154 @@ export function SpanFlyout({
           )}
         </EuiFlyoutHeader>
         <EuiFlyoutBody>
-          <StickySpanProperties span={span} transaction={parentTransaction} />
-          <EuiSpacer size="m" />
-          <Summary
-            items={[
-              <TimestampTooltip time={span.timestamp.us / 1000} />,
-              <>
-                <DurationSummaryItem
-                  duration={span.span.duration.us}
-                  totalDuration={totalDuration}
-                  parentType="transaction"
-                />
-                {span.span.composite && (
-                  <CompositeSpanDurationSummaryItem
-                    count={span.span.composite.count}
-                    durationSum={span.span.composite.sum.us}
-                  />
-                )}
-              </>,
-              <ContainerWithMarginRight>
-                {spanHttpUrl && (
-                  <HttpInfoSummaryItem
-                    method={spanHttpMethod}
-                    url={spanHttpUrl}
-                    status={spanHttpStatusCode}
-                  />
-                )}
-                <EuiToolTip
-                  content={i18n.translate(
-                    'xpack.apm.transactionDetails.spanFlyout.spanType',
-                    { defaultMessage: 'Type' }
-                  )}
-                >
-                  <EuiBadge color="hollow">{spanTypes.spanType}</EuiBadge>
-                </EuiToolTip>
-                {spanTypes.spanSubtype && (
-                  <EuiToolTip
-                    content={i18n.translate(
-                      'xpack.apm.transactionDetails.spanFlyout.spanSubtype',
-                      { defaultMessage: 'Subtype' }
-                    )}
-                  >
-                    <EuiBadge color="hollow">{spanTypes.spanSubtype}</EuiBadge>
-                  </EuiToolTip>
+          {isLoading && <EuiLoadingContent />}
```

Comment:
Why not just show this up front, and then avoid having to check if span is defined?